pyproject.toml

```diff
@@ -7,16 +7,44 @@ build-backend="setuptools.build_meta"
 
 [tool.black]
 skip-string-normalization = false
-exclude = '''
+include = '''
+\.pyi?$
+| ^/agent/bench-scripts/postprocess/user-benchmark-wrapper$
+| ^/agent/bench-scripts/test-bin/mock-postprocess/dbench-postprocess$
+| ^/agent/bench-scripts/test-bin/mock-postprocess/uperf-postprocess$
+| ^/agent/bench-scripts/test-bin/java$
+| ^/agent/tool-scripts/datalog/haproxy-ocp-datalog$
```

Comment:
Well, dang; this makes the convention of `xxx.py` and softlink `xxx` look really good, because I hate to see all these special cases. I wish we could at least filter on something like `$(file ${name}|grep 'Python script')`

Reply:
All ears if somebody can figure out how to get `black` to do that.

Reply:
We could consider consolidating this list by including directories, rather than specific files.

Or, we can wrap Black in a script which generates the value for the `--include` switch, using `find`, `file`, and `grep`.

And/or...we can reorganize our source tree so that certain directories only contain Python files (regardless of how they are named), and let Black loose on those without restrictions.

Reply:
Seems best to address that level of work in a future PR.  Are you interested in pursuing that?